suggestions/urls.py

```diff
@@ -11,4 +11,7 @@
     ),
     path(r"<int:pk>/", views.ProblemSuggestionUpdate.as_view(), name="suggest-update"),
     path(r"list/", views.ProblemSuggestionList.as_view(), name="suggest-list"),
+    path(
+        r"queue/", views.SuggestionQueueList.as_view(), name="suggestion-queue-listing"
```

Comment:
Seems it'd be better to name it `suggest-queue-listing` to be consistent with the other URL's